src/app/components/OptimoPromos/Lists/index.jsx

```diff
@@ -0,0 +1,21 @@
+import { GEL_SPACING_DBL } from '#app/legacy/gel-foundations/src/spacings';
```

Comment:
In the standards `Put styled components in a file with the same filename as the component but with .styled before the .jsx file extension.`

Reply:
Renamed the file